### ISSUE TITLE
Address Field throwing an error

### ISSUE THREAD

**Issue by ljclark** [verbb/formie#878]
**Description**

When adding an address field into a form, it's throwing the attached stack trace error when the page loads. 

This is on a multi-page form (page 1). The address field is the default field when selected from the UI, no customisations.

Removing the field resolves the issue and the form works as expected. 

```php
An Error occurred while handling another error:
TypeError: craft\helpers\Typecast::properties(): Argument #2 ($properties) must be of type array, null given, called in /project-xyz/vendor/craftcms/cms/src/base/Model.php on line 69 and defined in /project-xyz/vendor/craftcms/cms/src/helpers/Typecast.php:44
Stack trace:
#0 /project-xyz/vendor/craftcms/cms/src/base/Model.php(69): craft\helpers\Typecast::properties('verbb\\formie\\mo...', NULL)
#1 /project-xyz/vendor/verbb/formie/src/fields/formfields/Address.php(196): craft\base\Model->__construct(NULL)
#2 /project-xyz/vendor/craftcms/cms/src/base/Element.php(4661): verbb\formie\fields\formfields\Address->normalizeValue(NULL, Object(verbb\formie\elements\Submission))
#3 /project-xyz/vendor/craftcms/cms/src/base/Element.php(3653): craft\base\Element->normalizeFieldValue('address')
#4 /project-xyz/vendor/craftcms/cms/src/base/Element.php(1909): craft\base\Element->getFieldValue('address')
#5 /project-xyz/vendor/verbb/formie/src/base/FormFieldTrait.php(564): craft\base\Element->__get('address')
#6 /project-xyz/vendor/twig/twig/src/Extension/CoreExtension.php(1607): verbb\formie\base\FormField->getFieldValue(Object(verbb\formie\elements\Submission))
#7 /project-xyz/vendor/craftcms/cms/src/helpers/Template.php(101): twig_get_attribute(Object(craft\web\twig\Environment), Object(Twig\Source), Object(verbb\formie\fields\formfields\Address), 'getFieldValue', Array, 'method', false, false)
#8 /project-xyz/storage/runtime/compiled_templates/d1/d1deac972f59e6ba03bdd522b574ae9eefbb78e01932f5c436abcb352fea2805.php(63): craft\helpers\Template::attribute(Object(craft\web\twig\Environment), Object(Twig\Source), Object(verbb\formie\fields\formfields\Address), 'getFieldValue', Array, 'method')
#9 /project-xyz/vendor/twig/twig/src/Template.php(394): __TwigTemplate_dcb9b770951498775f5e152422289d709bc4f2762090dc5490ca4653b4983e8a->doDisplay(Array, Array)
#10 /project-xyz/vendor/twig/twig/src/Template.php(367): Twig\Template->displayWithErrorHandling(Array, Array)
#11 /project-xyz/vendor/twig/twig/src/Template.php(379): Twig\Template->display(Array)
#12 /project-xyz/vendor/twig/twig/src/TemplateWrapper.php(40): Twig\Template->render(Array, Array)
#13 /project-xyz/vendor/twig/twig/src/Environment.php(277): Twig\TemplateWrapper->render(Array)
#14 /project-xyz/vendor/craftcms/cms/src/web/View.php(414): Twig\Environment->render('formie/_special...', Array)
#15 /project-xyz/vendor/verbb/formie/src/elements/Form.php(1281): craft\web\View->renderTemplate('formie/_special...', Array, 'cp')
#16 /project-xyz/vendor/verbb/formie/src/services/Rendering.php(221): verbb\formie\elements\Form->renderTemplate(Array, Array)
#17 /project-xyz/vendor/verbb/formie/src/variables/Formie.php(145): verbb\formie\services\Rendering->renderField(Object(verbb\formie\elements\Form), Object(verbb\formie\fields\formfields\Address), NULL)
#18 /project-xyz/vendor/twig/twig/src/Extension/CoreExtension.php(1607): verbb\formie\variables\Formie->renderField(Object(verbb\formie\elements\Form), Object(verbb\formie\fields\formfields\Address), NULL)
#19 /project-xyz/vendor/craftcms/cms/src/helpers/Template.php(101): twig_get_attribute(Object(craft\web\twig\Environment), Object(Twig\Source), Object(verbb\formie\variables\Formie), 'renderField', Array, 'method', false, false)
#20 /project-xyz/storage/runtime/compiled_templates/87/8726ccd4a0ac5b36a0c72dbb0942084dd9521e33e1c4a8c30e9fddcf9bdbd91e.php(95): craft\helpers\Template::attribute(Object(craft\web\twig\Environment), Object(Twig\Source), Object(verbb\formie\variables\Formie), 'renderField', Array, 'method')
#21 /project-xyz/vendor/twig/twig/src/Template.php(394): __TwigTemplate_dfc22c3a48df1096738703b8281f00f6a2e42e13ffc716fe2d7f369b17d96e26->doDisplay(Array, Array)
#22 /project-xyz/vendor/twig/twig/src/Template.php(367): Twig\Template->displayWithErrorHandling(Array, Array)
#23 /project-xyz/vendor/twig/twig/src/Template.php(379): Twig\Template->display(Array)
#24 /project-xyz/vendor/twig/twig/src/TemplateWrapper.php(40): Twig\Template->render(Array, Array)
#25 /project-xyz/vendor/twig/twig/src/Environment.php(277): Twig\TemplateWrapper->render(Array)
#26 /project-xyz/vendor/craftcms/cms/src/web/View.php(414): Twig\Environment->render('formie/_special...', Array)
#27 /project-xyz/vendor/verbb/formie/src/elements/Form.php(1281): craft\web\View->renderTemplate('formie/_special...', Array, 'cp')
#28 /project-xyz/vendor/verbb/formie/src/services/Rendering.php(164): verbb\formie\elements\Form->renderTemplate(Array, Array)
#29 /project-xyz/vendor/verbb/formie/src/variables/Formie.php(128): verbb\formie\services\Rendering->renderPage(Object(verbb\formie\elements\Form), Object(verbb\formie\models\FieldLayoutPage), NULL)
#30 /project-xyz/vendor/twig/twig/src/Extension/CoreExtension.php(1607): verbb\formie\variables\Formie->renderPage(Object(verbb\formie\elements\Form), Object(verbb\formie\models\FieldLayoutPage), NULL)
#31 /project-xyz/vendor/craftcms/cms/src/helpers/Template.php(101): twig_get_attribute(Object(craft\web\twig\Environment), Object(Twig\Source), Object(verbb\formie\variables\Formie), 'renderPage', Array, 'method', false, false)
#32 /project-xyz/storage/runtime/compiled_templates/36/36790f35230c638b5286913626494e74e2ff03c46fb0d53700b0ed91258e0380.php(257): craft\helpers\Template::attribute(Object(craft\web\twig\Environment), Object(Twig\Source), Object(verbb\formie\variables\Formie), 'renderPage', Array, 'method')
#33 /project-xyz/vendor/twig/twig/src/Template.php(394): __TwigTemplate_45f606a3021bc66d18c8ad263ab65dff236d387eb6fe94e9ec88aa62702b6162->doDisplay(Array, Array)
#34 /project-xyz/vendor/twig/twig/src/Template.php(367): Twig\Template->displayWithErrorHandling(Array, Array)
#35 /project-xyz/vendor/twig/twig/src/Template.php(379): Twig\Template->display(Array)
#36 /project-xyz/vendor/twig/twig/src/TemplateWrapper.php(40): Twig\Template->render(Array, Array)
#37 /project-xyz/vendor/twig/twig/src/Environment.php(277): Twig\TemplateWrapper->render(Array)
#38 /project-xyz/vendor/craftcms/cms/src/web/View.php(414): Twig\Environment->render('formie/_special...', Array)
#39 /project-xyz/vendor/verbb/formie/src/elements/Form.php(1281): craft\web\View->renderTemplate('formie/_special...', Array, 'cp')
#40 /project-xyz/vendor/verbb/formie/src/services/Rendering.php(94): verbb\formie\elements\Form->renderTemplate(Array, Array)
#41 /project-xyz/vendor/verbb/formie/src/variables/Formie.php(110): verbb\formie\services\Rendering->renderForm(Object(verbb\formie\elements\Form), NULL)
#42 /project-xyz/vendor/twig/twig/src/Extension/CoreExtension.php(1607): verbb\formie\variables\Formie->renderForm(Object(verbb\formie\elements\Form))
#43 /project-xyz/vendor/craftcms/cms/src/helpers/Template.php(101): twig_get_attribute(Object(craft\web\twig\Environment), Object(Twig\Source), Object(verbb\formie\variables\Formie), 'renderForm', Array, 'method', false, false)
#44 /project-xyz/storage/runtime/compiled_templates/dc/dccfc1a043a837f9bbb0e8c7e5dad462e24e4e7f31a213453305667d7354c231.php(200): craft\helpers\Template::attribute(Object(craft\web\twig\Environment), Object(Twig\Source), Object(verbb\formie\variables\Formie), 'renderForm', Array, 'method')
#45 /project-xyz/vendor/twig/twig/src/Template.php(394): __TwigTemplate_1ce72737b3974cccb3e2ee4a1276e56ac027cab174decfbcbcf9aa9027613d30->doDisplay(Array, Array)
#46 /project-xyz/vendor/twig/twig/src/Template.php(367): Twig\Template->displayWithErrorHandling(Array, Array)
#47 /project-xyz/vendor/twig/twig/src/Template.php(379): Twig\Template->display(Array)
#48 /project-xyz/vendor/twig/twig/src/TemplateWrapper.php(40): Twig\Template->render(Array, Array)
#49 /project-xyz/vendor/twig/twig/src/Environment.php(277): Twig\TemplateWrapper->render(Array)
#50 /project-xyz/vendor/craftcms/cms/src/web/View.php(414): Twig\Environment->render('', Array)
#51 /project-xyz/vendor/craftcms/cms/src/web/View.php(467): craft\web\View->renderTemplate('', Array)
#52 /project-xyz/vendor/craftcms/cms/src/web/TemplateResponseFormatter.php(52): craft\web\View->renderPageTemplate('', Array, 'site')
#53 /project-xyz/vendor/yiisoft/yii2/web/Response.php(1095): craft\web\TemplateResponseFormatter->format(Object(craft\web\Response))
#54 /project-xyz/vendor/craftcms/cms/src/web/Response.php(275): yii\web\Response->prepare()
#55 /project-xyz/vendor/yiisoft/yii2/web/Response.php(339): craft\web\Response->prepare()
#56 /project-xyz/vendor/yiisoft/yii2/web/ErrorHandler.php(136): yii\web\Response->send()
#57 /project-xyz/vendor/craftcms/cms/src/web/ErrorHandler.php(185): yii\web\ErrorHandler->renderException(Object(TypeError))
#58 /project-xyz/vendor/yiisoft/yii2/base/ErrorHandler.php(135): craft\web\ErrorHandler->renderException(Object(TypeError))
#59 /project-xyz/vendor/craftcms/cms/src/web/ErrorHandler.php(64): yii\base\ErrorHandler->handleException(Object(TypeError))
#60 [internal function]: craft\web\ErrorHandler->handleException(Object(TypeError))
#61 {main}
```

**Steps to reproduce**
1. Create a multipage form
2. Add in several fields (this particular form has about 20 other fields in it).
3. Add in an address field
4. Try to render on the front end. 

### Form settings
Please pick from the options below.

- Multi-page form: Yes
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 2.0.0-beta.6
- Craft version: Craft Pro 4.0.1
- Multi-site: No

**Additional context**

This is a very vanilla install, no other plugins other than Formie are used.

**Comment by ljclark** [comment]
@engram-design  For some further context on this, i've got other form notifications working, but I'm only able to replicate this error when a 'Table' field is used in the form, so I'm assuming it's tied to that. 

If I remove the conditional from `{% for option in col.options if option.value == cellValue %}` to `{% for option in col.options %}` (Line 35 - vendor/verbb/formie/src/templates/_special/email-template/fields/table.html), it removes the error in the preview window, but it's not actually outputting any table values in the email notifications, but does let the email send! 

Similarly with the Repeater field, I can't get the values of the repeater fields to render in the preview email, it just skips over them.

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.0-beta.6",
  "...": "..."
}
```

Then run `composer update`.

**Comment by ljclark** [comment]
Awesome, thanks @engram-design - confirmed that's fixed it! 🎉

**Comment by engram-design** [comment]
Fixed in [2.0.0-beta.7](https://github.com/verbb/formie/releases/tag/2.0.0-beta.7)
